Applications/Spire/Source/KeyBindings/TaskKeyBindingsTableModel.cpp

```diff
@@ -14,43 +14,44 @@ namespace {
   const auto COLUMN_COUNT = 9;
 
   QVariant to_variant(const any& value) {
-    if(value.type() == typeid(Quantity)) {
+    const auto& type = value.type();
```

Comment:
If `value.type()` returns a reference (const or non-const), we normally bind it to `auto&`. Binding non-const references to a non-const reference may seem to violate const-correctness, but I think we do it for brevity.
If `value.type()` returns a value, you shouldn't bind it to a const reference, mainly to prevent confusion. Assign as in:
`auto type = value.type();`
in that case. Finally, even if `value.type()` return a reference, but the object reference is of pointer size or less, you should still prefer copying the object, same as passing `int` instead of `const int&` to a function.